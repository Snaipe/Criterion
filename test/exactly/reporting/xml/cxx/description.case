[act]

% description.cc.bin --xml

[assert]

stdout
  -transformed-by ( replace_version | replace_times )
  equals -contents-of expected/description.txt
