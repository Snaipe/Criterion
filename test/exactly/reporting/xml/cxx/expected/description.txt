<?xml version="1.0" encoding="UTF-8"?>
<!-- Tests compiled with Criterion <VERSION> -->
<testsuites name="Criterion Tests" tests="2" failures="1" errors="0" disabled="1">
  <testsuite name="misc" tests="2" failures="1" errors="0" disabled="1" skipped="1" time="<TIME>">
    <testcase name="skipped" assertions="0" status="SKIPPED" time="<TIME>">
      <skipped/>
    </testcase>
    <testcase name="failing" assertions="1" status="FAILED" time="<TIME>">
      <failure type="assert" message="1 assertion(s) failed."><![CDATA[description.cc:4: The expression 0 is false.]]>&#10;</failure>
    </testcase>
  </testsuite>
</testsuites>
